### PR TITLE
feat: home page

### LINKED ISSUE #5: Home Page
As a developer, 
I want see home page, 
so that I can explore home page content.

### In scope
- implement routing
- think about page content
- update figma and wiki text if necessary
- implement tablet and mobile versions
- unit test

### Acceptance Criteria

Scenario A: See Home page content

Given that local server is running, 
when navigate to root route, 
then home page is loaded.

Scenario B: Header Links

Given that local server is running, 
when navigate to root route,
then header links is loaded
and when I click
then redirect to appropriate page.

### Out of scope
- production server

### Resources:

* Figma: https://www.figma.com/file/jRZ9Rde5hBU2Ujitx1lQjX/Books-club?node-id=0%3A1&t=CvApC3btELgVe8cO-0
* Wiki: https://github.com/AnhelinaVr/in-my-book-frontend/wiki/Functional-Requirements#22-home-page

### Notes

- think about deployment to production server

### PR DESCRIPTION
**Pull request by AnhelinaVr**
### Task closes #5 
TODO: 
- mui types
- font

* [ ] `npm run lint` check passed without any failures or warnings
* [x] functionality is covered by new tests
* [x] functionality works in production build
* [x] existing functionality remains unchanged
* [ ] updated documentation (if required)

### CODE REVIEW

**Review by OlyaPolya — APPROVED** [review]
Code looks nice and neat

**Review by Konstantsin-Shumak — APPROVED** [review]

File: src/components/App/index.tsx [hunk 1]
```diff
@@ -1,43 +1,32 @@
...
+  { path: Paths.home, name: 'Home' },
+  { path: Paths.home, name: 'Rooms' },
+  { path: Paths.home, name: 'Books' },
+];
 
 const App = () => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  // TODO: alert wrapper
```
Comment (AnhelinaVr) [inline src/components/App/index.tsx:19]: Create a task

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <Link href={link.path} key={link.name} sx={styles.link}>
+            {link.name}
+          </Link>
+        ))}
+      </Box>
+      {isAuthenticated ? (
+        <Profile />
+      ) : (
+        <Box sx={styles.registrationContainer}>
+          {/* TODO: button hovers */}
```
Comment (AnhelinaVr) [inline src/components/Header/index.tsx]: Remove

File: src/constants/index.ts [hunk 3]
```diff
@@ -1 +1,9 @@
-export {};
+export const Paths = {
+  home: '/',
```
Comment (AnhelinaVr) [inline src/constants/index.ts:2]: Specify application paths

File: src/pages/Home/components/Cards/styles.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    fontWeight: theme.typography.fontWeightBold,
+    textDecoration: 'underline',
+    textUnderlineOffset: theme.spacing(2),
+    textDecorationColor: theme.palette.background.darker,
+    textDecorationThickness: 1.5,
+    marginBottom: theme.spacing(4),
+  },
+  image: {
+    maxWidth: theme.spacing(50),
+    flex: 1, // TODO: image heights are equal?
```
Comment (AnhelinaVr) [inline src/pages/Home/components/Cards/styles.ts]: Remove todo

File: package.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test --watchAll=false",
+    "test": "react-scripts test --watchAll=false -- --config=jest.config.json",
```
Comment (AnhelinaVr) [inline package.json]: "test": "jest"

**Pull request merged**